.github/workflows/ci.yml

```
@@ -567,7 +565,19 @@ jobs:
           name: pypi_files
           path: dist
 
-      - name: list dist files
+      - name: list dist files before PGO builds
+        run: |
+         ls -lh dist/
+         ls -l dist/
+         echo "`ls dist | wc -l` files"
+
+      - name: get PGO dist artifacts (comes after "get dist artifacts" to so these files override the non-PGO builds)
```

```suggestion
      - name: get PGO dist artifacts (comes after "get dist artifacts" so these files override the non-PGO builds)
```
typo and below, can be fixed another time.